app.js

```diff
@@ -1,11 +1,65 @@
-/* Imports */
-
+/* Imports and Get Dom Elements */
+const kanyeButtonEl = document.getElementById('kanye-button');
+const peteButtonEl = document.getElementById('pete-button');
+const doctorButtonEl = document.getElementById('doctor-button');
 /* Get DOM Elements */
 
+const kanyeContainer = document.getElementById('kanye-container');
+const peteContainer = document.getElementById('pete-container');
+const doctorContainer = document.getElementById('doctor-container');
+
+const correctGuessesEl = document.getElementById('wins');
+console.log(correctGuessesEl);
+const incorrectGuessesEl = document.getElementById('losses');
+const totalGuessesEl = document.getElementById('total');
+
 /* State */
+let correctGuesses = 0;
+let totalGuesses = 0;
 
 /* Events */
+kanyeButtonEl.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    displayResults('kanye', correctSpot);
+});
 
+peteButtonEl.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    displayResults('pete', correctSpot);
+});
+
+doctorButtonEl.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    displayResults('doctor', correctSpot);
+});
 /* Display Functions */
 
+function getRandomHidingSpot() {
+    const places = ['kanye', 'pete', 'doctor'];
+
+    const index = Math.floor(Math.random() * places.length);
+    const correctSpot = places[index];
+    return correctSpot;
+}
+
+function displayResults(userGuess, correctSpot) {
+    kanyeContainer.classList.remove('kim');
```

Comment:
This code is totally acceptable and works great, but these classList.remove functions could be moved into a new function maybe called resetFunction() or something. Then that function could be 'called' here on line 46. This strategy would 'clean up' your code and make it a bit easier to read.

Reply:
my question with the resetFunction() is what do I put in it? "(userGuess, correctSpot)" ?